Intersect (Core)/Config/PlayerOptions.cs

```diff
@@ -41,7 +41,7 @@ public class PlayerOptions
         /// Default value is 10. If you change the default value (e.g. 16 slots), make sure to properly
         /// modify the Client's HotbarWindow layout at: 'resources\gui\layouts\game\HotbarWindow.json'.
         /// </summary>
-        public int MaxHotbar = 10;
+        public int HotbarSlotCount = 10;
```

Comment:
This should be a property not a field

Reply:
Alright, will it do as AutoProperty? and how about the rest of PlayerOptions?

Reply:
Alright, will it do as AutoProperty?
Also, how about the rest of PlayerOptions fields?

Reply:
Yeah, realized about the difference afterwards, an Auto-property would made it a readonly-sort of property, which is not what we want here, since we want to be able to change this though the Json config files. Thanks for clarifying ! 👍🏽